src/pickup_meta.rs

```diff
@@ -329,7 +349,10 @@ pub struct RoomInfo
 {
     pub room_id: u32,
     pub name: &'static str,
+    pub name_id: u32,
+    pub mapa_id: u32,
```

Comment:
Same as `name_id` and `mapa_id` above.

Reply:
I am blocked on this until one of us figures out why resource_tracing is keying `PART` files as `CMDL`